client/src/components/Header.js

```diff
@@ -143,17 +143,18 @@ export default ({
   const languageMenu = (
     <Menu>
       {Object.entries(languages).map(([key, label]) => (
-        <Menu.Item id={GTM.nav.prefix + GTM.nav.language + GTM.language[key]} key={key}>
-          <a
+        <Menu.Item key={key}>
+          <span
```

Comment:
Okay I do see how this should solve it, moving the id to the same element as the event listener.

This would work except why change to `span`? The issue now is since it's `inline` you can't click on the menu item where there is no text.

![no-click-menu](https://user-images.githubusercontent.com/10546720/95872611-fa17e180-0d3c-11eb-8803-472a45c2acc7.gif)

We should either:

a) change `span` to an block element - either back to `a` or `div`
b) Move `id` back to `Menu.Item` **along with** event listener

Reply:
@Zaydme since the resolution was straight forward I went ahead. We'll merge to staging and then get the BI team to confirm all resolutions.

Reply:
@joshmorel 
a) After some testing I found out that GTM doesn't fire on <a> so had to change it to <span>
b) Moving id back to Menu.Item doesn't fire, because the click is being detected on the child <span>

Reply:
Thanks @Zaydme . Wasn't sure if you'd be able to do this til late today so I gave it a shot. b didn't work. I changed it to `div` so hopefully that is fine (@idjevm will confirm). We could also change style on span to display: block but I would be suprised if `span` worked and `div` didn't...